environments/aks/ptlsbox.tfvars

```diff
@@ -1,10 +1,10 @@
 clusters = {
-  "00" = {
-    kubernetes_cluster_version = "1.29"
-  },
-  # "01" = {
-  #   kubernetes_cluster_version = "1.25"
-  # }
+  # "00" = {
+  #   kubernetes_cluster_version = "1.29"
+  # },
+  "01" = {
+    kubernetes_cluster_version = "1.25"
```

Comment:
1.25?

Reply:
ignore that. it is not meant to be there

Reply:
yes it is on code.